api/src/test/java/org/openmrs/module/eptsreports/reporting/calculation/EptsCalculationUtilsMock.java

```diff
@@ -0,0 +1,31 @@
+package org.openmrs.module.eptsreports.reporting.calculation;
+
+import mockit.Mock;
+import mockit.MockUp;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
+
+public class EptsCalculationUtilsMock extends MockUp<EptsCalculationUtils> {
```

Comment:
These helper functions can be combined with the helper function `createBasicObs` from CalculationsTestsCache into the new OpenMRSTestHelper(named for example)  class since these are specific to obs and encounters etc...

In the long run we might want to have a file per type (obs/encounter/etc... but lets wait till that becomes a need)

Reply:
those mocks however are meant to replace their respective methods and are better kept in separate mock files as we have done, in one review up you suggested these be invoked from BasePatientCalculationTest which makes sense and so i only have createBasicObs in OpenMRSTestHelper

Reply:
Ah I see, my mistake. I misread what these are. I mistook these for utils of the mock classes, for some reason a read over the mock annotations! :man_facepalming: 

As you were :smile: